content/docs/components-and-props.md

```diff
@@ -16,57 +16,61 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+Komponen mempermudah Anda untuk memecah antarmuka pengguna menjadi bagian tersendiri, bagian yang bisa digunakan kembali, dan berpikir tentang setiap potongan dalam isolasi.
```

Comment:
`You can find a [detailed component API reference here](/docs/react-component.html).` line is still waiting for the translation 😄

Reply:
Hi, sorry for late response. I have resolved the conflict and finishing translate for this missing part. Thanks for give me a notice 😄